### ISSUE TITLE
Typo in docs

### ISSUE THREAD

**Issue by drizt** [OpenShot/openshot-qt#685]
Here http://openshot.org/static/files/user-guide/titles.html

    3 	Save 	Save and add the title to your project

But must be 

    4 	Save 	Save and add the title to your project

**Comment by peanutbutterandcrackers** [comment]
@drizt - Good sir, could you please do a typo-rectification Pull Request? OpenShot is being developed by a single man. And he's already under way too much pressure. If you and I don't lend a helping hand, I wonder who will...

**Comment by peanutbutterandcrackers** [comment]
@ferdnyc - could you perhaps also fix this typo in your PR? (Perhaps resubmit a PR with this typo fixed?) Every single possible bug fix counts. :)

**Comment by peanutbutterandcrackers** [comment]
@drizt: Haha, thank you for the PR! Every small help counts. But I find it funny that you submitted just that typo-fix. lol I thought you were going to go through the whole doc and fix every single typo and then submit the PR. :smile: 

But please do keep on making other PRs too. Every single help counts! I see that you have some C++ repos too. And also a member of Russian Fedora. I'm sure you can help fix a lot of other bugs too. Please do continue to support OpenShot, comrade! Let's make OpenShot better together! :+1:

**Comment by ferdnyc** [comment]
Every PR _does_ help, but at the same time multiple PRs changing the same files isn't a great situation, because now we have a bit of a race condition. My docs PR won't apply cleanly after @drizt 's is merged, and vice versa, so somebody's going to have to babysit their PR.

That's why, honestly, I'm not that into the idea of a lot of PRs piling up. I'm happy to help out. But only if someone's merging.

**Comment by ferdnyc** [comment]
(Actually, I may be wrong about that. If we're lucky and the table in question was one I _didn't_ have to modify, then the two PRs shouldn't conflict. I edited about half of the tables, so it's a 50/50 shot. But the larger point still stands.)

**Comment by peanutbutterandcrackers** [comment]
@ferdnyc - I am also waiting on Mr. Thomas to come back. I often feel like he's doing [this](https://www.youtube.com/watch?v=nL8hVXSDmNM) when he's not around - not answering any emails or not making any commits. But in the end, he's the one who made OpenShot what it is today and I am willing to wait till he shows up.... I know it sucks. Sorry about that... but I'm sure Mr. Thomas will be super happy when he actually sees the PR. :+1:

**Comment by ferdnyc** [comment]
I mean, it's cool and completely understandable — we all get busy. And there's certainly no rush; heck, it's less work for me, to just keep whatever other stuff I've got in mind on hold for now, and return to it later. I'm not going anywhere. 

What I'm saying, I guess, is that my impression is that this isn't a good time to be submitting new changes to the OpenShot tree. So, rather than have a bunch of work back up in the PR queue, which (for reasons I explained) tends to be problematic, I prefer to hold off for now, and let what's already been submitted stand on its own until resources are available on the maintainer end. Once the existing backlog gets dealt with and the queue is moving again (and after there's been a chance to exchange feedback, if needed, so that I can make any adjustments necessary to ensure that I'm helping _correctly_ and in a way that's genuinely useful), then and only then would it make sense [to me] to start pushing additional work onto the queue.
